src/requests/playbook.rs

```diff
@@ -18,4 +18,13 @@ use utoipa::ToSchema;
 #[derive(Debug, Serialize, Deserialize, ToSchema)]
 pub struct CreatePlaybookRequest {
     pub repo: String,
+    pub reference: Option<String>,
+}
+pub trait GitReferenceMethods {
+    fn new(repo: String, branch: Option<String>) -> Self;
+}
+impl GitReferenceMethods for amp_common::schema::GitReference {
+    fn new(repo: String, branch: Option<String>) -> Self {
+        amp_common::schema::GitReference { repo, branch, ..Default::default() }
+    }
 }
```

Comment:
It do not be used more times, it's a temporary function, why not new a GitRef object with attributes directly?